ntc_templates/templates/cisco_ios_show_wireless_tag_policy_summary.textfsm

```diff
@@ -0,0 +1,21 @@
+Value Required NAME (\S+)
+Value DESCRIPTION (.+)
+
+Start
+  # Skip the header lines
+  ^Number of Policy Tags:.*
+  ^Policy Tag Name\s+Description
```

Comment:
```suggestion
  ^Policy Tag Name\s+Description\s*$$
```
I like to ensure the column headers are what we expect, so allow for space and end of line